src/domain/accounts/services/Cargo.toml

```diff
@@ -23,12 +23,15 @@ doctest = false
 
 [dependencies]
 database-common = { workspace = true }
+email-gateway = { workspace = true }
```

Comment:
This seems like an odd coupling. I expected that in account domain layer we only emit messages about account life cycle and react to them elsewhere.

The fact that these messages trigger an email notification is IMO neither a account domain concern, nor even an infra-level account service concern. Sending emails notifs is more like Kamu Node-level business logic concern.

I would suggest moving the templating and email sending into a crate in `kamu-node`. Seems like one of those things that we'll never need on CLI level.

Reply:
It's a good point. It's there, as is there was no other place to hold it. I guess I will start a Kamu-node branch pointing on this branch.